ironfish/src/network/peerNetwork.ts

```diff
@@ -632,9 +631,7 @@ export class PeerNetwork {
   ): Promise<void> {
     const { message } = incomingMessage
 
-    if (message instanceof GossipNetworkMessage) {
-      await this.handleGossipMessage(peer, message)
```

Comment:
If we're no longer going to parse Gossip messages, we should delete them and refactor the code to remove references, though it'll require some changes to `onNewTransaction`+more.

Reply:
Yea I can make that change in this PR. I was thinking of doing it in a follow up just so the PRs are easier to review. I think they can be logically separated between receiving old messages and sending old messages wdyt?

Reply:
Sounds good, as long as the rest of the work is in progress (and it sounds like it's almost done) I'm fine with that